sections/discussion/discussion-chapter.tex

```diff
@@ -1,9 +1,153 @@
+For the two other datasets, both methods give very mediocre results.
+Finally, for the NDCG metric, we see that the \textit{IS-Userbased} performs very well on the Yahoo dataset, mediocre on LDOS-CoMoDa and poorly on MovieLens.\\\\
+\\\\
+The Yahoo dataset does not contain timestamps, nor any other contextual information.
+It includes age of the user and their gender, which are considered to be user attributes rather than contextual information.
+As such, this dataset only contains side information.
+\autoref{subsec:desc-of-datasets} described how this side information was used as contextual variables for context-aware methods.
+This could influence the results on this datasets for the methods utilizing context.
+However, on the results we see generally poorer performance all methods on the Yahoo dataset with the exception of \textit{IS-UserBased}, but not substantially.
+For the context-aware methods, we do not see a decrease in performance compared to the other methods, indicating that utilizing this side-information as context does not negatively impact the results.
+An explanation for the worse performance is likely the increased sparsity of the Yahoo dataset, making better results more difficult to achieve.\\
+To check whether using the side-information in contextual methods made a difference, we re-ran the the \textit{CAMF-C(I)} and \textit{IS-UserBased} methods on the MovieLens dataset using respectively contextual (timestamp) and side-information (age and gender) for the models.
+\begin{table*}[!htp]\centering
+    \caption{Results for using respectively context and side-information for the MovieLens dataset, highlighted numbers are the best results}\label{tab:movielenscontextsideinfo}
+    \scriptsize
+    \begin{tabular}{lrrrrrrrr}\toprule
+    &\textbf{RMSE} &\textbf{MAE} &\textbf{Precision@10} &\textbf{Recall@10} &\textbf{MAP@10} &\textbf{NDCG} &\textbf{F1@10} \\\cmidrule{2-8}
+    \textbf{CAMF-C Context} &0.931 &0.738 &\textbf{0.726} &0.645 &0.0025 &0.375 &\textbf{0.683} \\\cmidrule{1-8}
+    \textbf{CAMF-CI Context} &0.978 &0.773 &0.713 &0.628 &0.0007 &0.403 &0.668 \\\cmidrule{1-8}
+    \textbf{CAMF-C Side-info} &\textbf{0.930} &\textbf{0.736} &0.726 &0.646 &0.0026 &0.376 &0.684 \\\cmidrule{1-8}
+    \textbf{CAMF-CI Side-info} &0.952 &0.753 &0.714 &0.630 &0.0005 &0.403 &0.669 \\\cmidrule{1-8}
+    \textbf{IS-UserBased Side-info} &1.329 &0.976 &0.536 &\textbf{0.789} &\textbf{0.0220} &\textbf{0.538} &0.638 \\\cmidrule{1-8}
+    \textbf{IS-UserBased Context} &1.366 &1.041 &0.344 &0.785 &0.0120 &0.270 &0.478 \\\cmidrule{1-8}
+    \bottomrule
+    \end{tabular}
+\end{table*}
+As seen on \autoref{tab:movielenscontextsideinfo}, the models generally perform better when provided with side-information rather than contextual information.
```

Comment:
```suggestion
As seen on \autoref{tab:movielenscontextsideinfo}, the model generally performs better when provided with side-information rather than contextual information.
```